tests/org.eclipse.n4js.ide.tests/probands/external_project_dependencies/P2/package.json

```diff
@@ -3,8 +3,8 @@
 	"version": "0.0.1",
 	"dependencies": {
 		"n4js-runtime": "*",
-		"react": "*",
-		"@n4jsd/react": "*"
+		"react": "17.0.1",
```

Comment:
Actually, this `*` was chosen deliberately to ensure that we are always compatible to the latest version.